src/main/java/beam/analysis/plots/DeadHeadingAnalysis.java

```diff
@@ -168,9 +170,10 @@ private void processDeadHeadingDistanceRemainingRepositionings() {
                 }
             }
         }
-        finally {
-            vehicleEvents.clear();
+        catch (Exception e) {
```

Comment:
Changing the finally's to catch's is a behavior change. Now it will swallow exceptions, whereas before it didn't. Should it rethrow, call reset, swallow (as here), or something else?